test/verify/check-docker

```diff
@@ -25,6 +25,20 @@ import os
 
 @skipImage("No cockpit-docker on i386", "fedora-i386")
 class TestDocker(MachineCase):
+    def setUp(self):
+        MachineCase.setUp(self)
+        m = self.machine
+
+        # HACK: https://github.com/docker/docker/issues/25246
+        if m.image in [ "debian-8", "debian-testing" ]:
+            m.execute("printf '\\n\\n[Service]\\nKillMode=process\\n' >> /lib/systemd/system/docker.service")
+            m.execute("systemctl daemon-reload")
+
+        m.execute("systemctl start docker")
+
+        # HACK: docker fails to start a container without /etc/resolv.conf
+        m.execute("touch /etc/resolv.conf")
```

Comment:
Personally I'd much rather leave this hack in the tests (it's cheap to do) rather than "polluting" the images with it, and this is a bug which should eventually be solved in docker proper.